plugins/prettier/.toolkitrc.yml

```diff
@@ -6,8 +6,9 @@ commands:
   'format:staged': Prettier
 
 options:
-  - PackageJson:
-      scripts:
-        format: 'format:local'
```

Comment:
**thought**: oops. we should validate that `options` doesn't have any extraneous keys and warn about it

Reply:
yeah we can probably add a zod validation step after the substitution step. and maybe do an integration test with all our plugins